cypress/support/fragments/inventory/fastAddNewRecord.js

```diff
@@ -0,0 +1,120 @@
+import {
+  MultiColumnListCell,
+  Button,
+  Select,
+  Selection,
+  SelectionList,
+  TextField,
+  KeyValue,
+  Section,
+} from '../../../../interactors';
+import getRandomPostfix from '../../utils/stringTools';
+
+const setDefaultInstanceStatus = (statusCode) => {
+  cy.do(Select({ name: 'instanceStatusCode' }).choose(statusCode));
+};
+
+const saveFastAddSettings = () => {
+  cy.do(Button('Save').click());
+};
+
+const calloutMessages = {
+  INVENTORY_RECORDS_CREATE_SUCCESS: 'Inventory records have been created successfully',
+  SETTING_UPDATE_SUCCESS: 'Setting was successfully updated.'
+};
+
+const fastAddNewRecordFormDetails = {
+  defaultInstanceStatusCodeOption: 'Select instance status',
+  instanceStatusCodeOption: 'Uncataloged',
+  instanceStatusCodeValue: 'uncat',
+  resourceTitle: 'Monograph',
+  resourceType: 'text',
+  permanentLocationOption: 'Online (E) ',
+  permanentLocationValue: 'Online',
+  itemBarcode: `${getRandomPostfix()}Barcode`,
+  materialType: 'text',
+  permanentLoanType: 'Course reserves',
+  note: 'note for monograph',
+};
+
+const fillFastAddNewRecordForm = ({
+  resourceTitle,
+  resourceType,
+  permanentLocationOption,
+  itemBarcode,
+  materialType,
+  permanentLoanType,
+  note,
+}) => {
+  cy.do([
+    TextField('Resource title*').fillIn(resourceTitle),
+    Select('Resource type*').choose(resourceType),
+    Selection('Permanent location*').open(),
+    SelectionList().filter(permanentLocationOption),
+    SelectionList().select(permanentLocationOption),
+    TextField('Barcode').fillIn(itemBarcode),
+    Select('Material type*').choose(materialType),
+    Select('Permanent loan type*').choose(permanentLoanType),
+    TextField('Note*').fillIn(note),
+  ]);
+};
+
+const saveAndClose = () => {
+  cy.do(Button('Save and close').click());
+
+  cy.expect(Section({ id: 'pane-results' }).find(Button('Actions')).exists());
+};
+
+const openRecordDetails = (row = 0) => {
+  cy.do(MultiColumnListCell({ row, columnIndex: 1 }).click());
+
+  cy.expect(Section({ id: 'pane-instancedetails' }).exists());
+};
+
+const verifyRecordCreatedDate = ({ start, end }) => {
+  cy
+    .get('[class^="metaHeaderLabel"]')
```

Comment:
can we work though interactors?

Reply:
@dmitriyfolio. I found `MetaSection` interactor but it has no `actions` for extracting values. So, I had to go this way.